excel_import_export/README.rst

```diff
@@ -36,6 +36,7 @@ Without having to code to create excel file, developer do,
 - Design excel template using standard Excel application, e.g., colors, fonts, formulas, etc.
 - Instruct how the data will be located in Excel with simple dictionary instruction or from Odoo UI.
 - Odoo will combine instruction with excel template, and result in final excel file.
+- Functionality allowing you to add the excel file to an email template
```

Comment:
This module's functionality is optional to the main module (it adds a functionality that is not the main point of this module). If we want to stick close to the principle of 1 module = 1 scope = 1 functionality, it would be cleaner to split it into another module.